pandas/io/sql.py

```diff
@@ -572,8 +572,11 @@ def create(self):
         else:
             self._execute_create()
 
-    def insert_statement(self):
-        return self.table.insert()
+    def insert_statement(self, data, conn):
```

Comment:
can you add a doc-string here

Reply:
Could you add a parameters and returns section as well. http://numpydoc.readthedocs.io/en/latest/format.html